kolibri/core/apps.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriCoreConfig(AppConfig):
```

Comment:
My understanding is that Django opens a connection at the start of each request, and closes it at the end, unless you separately configure persistent connections.